### PR TITLE
added button width and margin to typographies

### PR DESCRIPTION
**Pull request by MeenuKumari28**
added button width and bottom margin to typographies
button
Earlier the button looked like `button` and after look of button is `primary-button`

### CODE REVIEW

File: form/button.ftd [hunk 1]
```diff
@@ -56,19 +56,21 @@
 caption title:
 optional string link:
 boolean $hover: false
+button-width width: default
 
 -- ftd.row:
```
Comment (amitu) [inline form/button.ftd:61]: Stop using ftd.row and use ds.row.

File: form/button.ftd [hunk 2]
```diff
@@ -56,19 +56,21 @@
...
 background.solid: $inherited.colors.cta-primary.base
 border-radius.px: 8
-padding-horizontal.px: 18
-padding-vertical.px: 16
+padding-horizontal.px: 16
+padding-vertical.px: 12
```
Comment (amitu) [inline form/button.ftd:65]: Use ds.spaces.* instead of hardcoded numbers like this.

**Comment by amitu** [comment]
@MeenuKumari28 on the button page, show the buttons, and include a screenshot in the PR description of before and after to make it easy to review changes.

**Comment by amitu** [comment]
@MeenuKumari28 so we have both button and primary button, but with different looks? Not found of this. @yashveer29 can you review the various kinds of buttons we have with Meenu, @MeenuKumari28 create a page with all different kinds of buttons, and include it in this PR only. Also include the screenshot of that page here so Yashveer can see.

**Comment by MeenuKumari28** [comment]
@amitu all the button outputs are added inside [button](https://fastn-community.github.io/design-system/form/button/) and removed button component as we already have the primary button as default.
@yashveer29 output image 
button-snap

**Comment by amitu** [comment]
@MeenuKumari28, good you removed `ds.button`. But `ds.button` was more "featureful" than `ds.primary-button` etc. Can you review if all the arguments to `ds.button` component are present on `ds.primary-button` etc as well.

**Comment by yashveer29** [comment]
@MeenuKumari28 do we have hover and clicked states for buttons ???

**Pull request merged**